ipn/ipnlocal/network-lock.go

```diff
@@ -238,6 +234,21 @@ func (b *LocalBackend) tkaSyncLocked(ourNodeKey key.NodePublic) error {
 	return nil
 }
 
+// tkaApplyDisablementLocked checks a disablement secret and locally disables
+// TKA (if correct). An error is returned if disablement failed.
+//
+// b.mu must be held & TKA must be initialized.
+func (b *LocalBackend) tkaApplyDisablementLocked(secret []byte) error {
+	if b.tka.authority.ValidDisablement(secret) {
```

Comment:
can `b.tka` be nil here?

Reply:
No, that codepath is guarded by `isEnabled` which itself is `b.tka != nil`